QMLComponents/components/JASP/Controls/JAGSTextArea.qml

```diff
@@ -6,4 +6,8 @@ TextArea
 {
 	textType: JASP.TextTypeJAGSmodel
 	showLineNumber: true
+	RSyntaxHighlighterQuick
```

Comment:
Isnt it a JASP.Controls.TextArea?
That one should have build-in support for syntaxhighlighting right?

Reply:
Nvm, that is apparently only a specialization